utils/filter_users.ts

```diff
@@ -2,19 +2,22 @@
 // See LICENSE.txt for license information.
 import {UserSearchOptions, UserListOptions, UserFilters} from 'utils/constants';
 
+export type FilterOptions = {
+    [key: string]: string | boolean;
+}
```

Comment:
Can you please include a `new-line` here before your comment for linting test to tass